src/rollup/RollupCreator.sol

```diff
@@ -114,6 +123,17 @@ contract RollupCreator is Ownable {
             })
         );
 
+        sequencerInbox.setIsBatchPoster(_batchPoster, true);
+
+        // Call setValidator on the newly created rollup contract
+        bool[] memory _vals = new bool[](_validators.length);
+        for (uint256 i = 0; i < _validators.length; i++) {
+            _vals[i] = true;
+        }
+        IRollupAdmin(address(rollup)).setValidator(_validators, _vals);
```

Comment:
We should add a check to not call setValidator if the validator list is empty, because it'll revert otherwise. We should also not set the _batchPoster if it's the zero address.

Reply:
is it enough to just check the lenght of validators @PlasmaPower ? Or need to check if all addresses are non-zero as well?